main.go

```diff
@@ -94,6 +96,24 @@ func scytale(arguments []string) int {
 
 	logger.Log(level.Key(), level.InfoValue(), "configurationFile", v.ConfigFileUsed())
 
+	u := v.Sub(traceProviderKey)
+	if u == nil {
+		fmt.Fprintf(os.Stderr, "traceProvider configuration is missing.\n")
+		return 1
+	}
+	config := &candlelight.Config{
+		ApplicationName: applicationName,
+	}
+	headerConfig := &candlelight.HeaderConfig{}
+	u.Unmarshal(headerConfig)
+	u.Unmarshal(config)
+	traceProvider, err := candlelight.ConfigureTracerProvider(*config)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to build traceProvider: %s\n", err.Error())
+		return 1
+	}
+	traceConfig := candlelight.TraceConfig{*headerConfig, traceProvider}
```

Comment:
Can you please fix this lint issue ("composite literal uses unkeyed fields")? It's talking about providing the key values for `TraceConfig`, which will allow this code to potentially still work if a new value is added in `TraceCofing` in the future.

Example fixed: `candlelight.TraceConfig{HeaderConfig:*headerConfig, TraceProvidere:traceProvider}`